### PR TITLE
Add License information

### PR DESCRIPTION
**Pull request by MatiasBjorling**
Add missing license information

Went with GPL-3. 

Removed the WD copyright notice, as it will be superseeded by the license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-Copyright (c) 2021 Western Digital Corporation or its affiliates.
```
Comment (damien-lemoal) [inline README.md:1]: Arg, no. License and copyright are 2 different things. Adding the license file does not mandate remove the copryright. Please keep it.

File: README.md [hunk 2]
```diff
@@ -74,7 +72,8 @@ $ make upgrade
...
 ## Contributing
 
-Read the [CONTRIBUTING](CONTRIBUTING) file and use GitHub pull request to
-contribute changes and improvements.
+Feel free to submit a pull request to contribute changes and improvements.
+
+## License
 
-For other inquiries, contact the zonedstorage.io team at zonedstorage@wdc.com.
+The site source code is released as open source software under the GPL v3 license, see the LICENSE file in the project root for the full license text.
```
Comment (damien-lemoal) [inline README.md:79]: Can you make LICENSE a link to the file ?

Comment (MatiasBjorling) [reply README.md:79]: Yep

Comment (damien-lemoal) [reply README.md:79]: I do not see "LICENSE" being a link... Make it [LICENSE](license.md) and that should do it.

File: README.md [hunk 3]
```diff
@@ -72,8 +72,10 @@ $ make upgrade
...
-Feel free to submit a pull request to contribute changes and improvements.
+Feel free to submit a pull request to contribute changes and improvements. 
 
-## License
+We follow the Contributor Covenant Code of Conduct as outline [here](CODE_OF_CONDUCT.md).
 
-The site source code is released as open source software under the GPL v3 license, see the LICENSE file in the project root for the full license text.
\ No newline at end of file
+## Copying and License
+
```
Comment (damien-lemoal) [inline README.md:80]: Extra white line added here.

Comment (damien-lemoal) [reply README.md:80]: otherwise looks good ! Fix and merge !

**Comment by MatiasBjorling** [comment]
Updated the pull request. Added a new patch. Also added copying file, and added the code of conduct.

**Comment by MatiasBjorling** [comment]
I think its possible to squash and merge the patchset. But doesn't seem to be enabled on the repo. Trying to locate the feature in settings.

**Pull request merged**